crates/runtime/src/runtime/default.rs

```diff
@@ -563,78 +562,106 @@ impl<T> Runtime for DefaultRuntime<T>
 where
     T: EnvTypes,
 {
-    fn validate_deploy(&self, bytes: &[u8]) -> std::result::Result<(), ValidateError> {
-        let template = self.env.parse_deploy(bytes, None)?;
+    fn validate_deploy(&self, message: &[u8]) -> std::result::Result<(), ValidateError> {
+        let template = self.env.parse_deploy(message, None)?;
```

Comment:
Good job on changing this variable name, we should start doing this more often. Rather than `bytes`, `slice`, let's use stuff such as `envelope`, `payload`, `wasm_module`, etc.. More descriptive stuff!